src/v2/guide/conditional.md

```diff
@@ -192,10 +192,8 @@ Generally speaking, `v-if` has higher toggle costs while `v-show` has higher ini
 When used together with `v-if`, `v-for` has a higher priority than `v-if`. See the <a href="../guide/list.html#v-for-with-v-if">list rendering guide</a> for details.
 
 
-### Truthiness/Falsiness with `v-bind`
+### Truthiness/Falsiness with `v-show`
```

Comment:
`Note that for other directives (e.g. `v-if` and `v-show`), JavaScript's normal truthiness still applies.`
previously version was correct